mne/report/report.py

```diff
@@ -458,9 +458,11 @@ def _iterate_trans_views(function, **kwargs):
     from ..viz.backends.renderer import backend
     try:
         try:
-            return _itv(function, fig, surfaces=['head-dense'], **kwargs)
+            return _itv(
+                function, fig, surfaces={'head-dense': 1.}, **kwargs
+            )
         except IOError:
-            return _itv(function, fig, surfaces=['head'], **kwargs)
+            return _itv(function, fig, surfaces={'head': 1.}, **kwargs)
```

Comment:
-1 on a totally opaque head, it makes it impossible to see dig points inside the head

Reply:
Are they still relevant at this stage? We state the distance between dig points and scalp in the caption. I thought it would be a good idea to see how the sensors are aligned in relation to the head.

But I suppose even the fiducial points could end up inside the head, which could be a big problem without transparency

Reply:
But do we agree that the current (in `main`) transparency levels are too … transparent? :)

![image](https://user-images.githubusercontent.com/2046265/146389006-5fd31ef1-0a5b-4519-aab9-567eba637f93.png)

Reply:
To me that looks okay

Reply:
Ok I'll revert this particular change then

Reply:
Reverted in df3e2e8